### PR TITLE
Add OTLP exporter README

### PR DESCRIPTION
**Pull request by nadiaciobanu**
Add a README to the OTLP exporter directory to describe the exporter and how it can be used/configured.

### CODE REVIEW

File: exporters/otlp/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# OTLP Exporter
+
+The [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/README.md) (OTLP) is a vendor-agnostic protocol designed as part of the OpenTelemetry project. The OTLP exporter can be used to export to any backend that supports OTLP.
+
+Currently, the only backend that supports OTLP is the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector). The Collector can be configured to export to many other backends, such as Zipkin and Jaegar.
```
Comment (reyang) [inline exporters/otlp/README.md:5]: Minor: I think we probably want to rephrase this to "The OpenTelemetry Collector is the reference implementation of OTLP backend" so we don't have to worry about whether it is the "only backend at this moment or in the future".

File: exporters/otlp/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+### Configuration options
+
+| Option       | Default           |
+| ------------ |------------------ |
+| `endpoint`   | `localhost:55680` |
+
+## Example
+For a complete example demonstrating how to use the OTLP exporter, see [examples/otlp](../../examples/otlp).
```
Comment (reyang) [inline exporters/otlp/README.md:25]: nit: probably add an empty line after level 2 heading - you can also try markdownlint.

**Comment by ZiweiZhao** [comment]
LGTM

**Comment by nadiaciobanu** [comment]
@reyang Ready to merge!
